### PR TITLE
Enable CI builds [WIP -- DO NOT MERGE YET, EVEN IF BUILDS PASS]

### PR DESCRIPTION
**Pull request by mikeage**

### CODE REVIEW

**Comment by mikeage** [comment]
(I'm going to want to squash this, so in case it takes a few tries, doing it as a PR to the target branch)

**Review by merulaxr — APPROVED** [review]
Tested most recent builds and they work! Tidyup in progress but good to merge after that.

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -774,7 +774,7 @@ PlayerSettings:
   additionalIl2CppArgs: 
   scriptingRuntimeVersion: 1
   gcIncremental: 0
-  assemblyVersionValidation: 1
```
Comment (merulaxr) [inline ProjectSettings/ProjectSettings.asset:777]: Shouldn't need to change this now!

Comment (mikeage) [reply ProjectSettings/ProjectSettings.asset:777]: Why not? What changed?

(I'll test without this)

Comment (mikeage) [reply ProjectSettings/ProjectSettings.asset]: huh, seems happy now. Not complaining :-)

**Pull request closed without merging**